webui/react/src/pages/WorkspaceDetails/WorkspaceProjects.tsx

```diff
@@ -139,17 +151,13 @@ const WorkspaceProjects: React.FC<Props> = ({ workspace, id, pageRef }) => {
   useEffect(() => {
     if (settings.whose === prevWhose || !settings.whose) return;
 
-    switch (settings.whose) {
-      case WhoseProjects.All:
-        updateSettings({ user: undefined });
-        break;
-      case WhoseProjects.Mine:
-        updateSettings({ user: user ? [user.id] : undefined });
-        break;
-      case WhoseProjects.Others:
-        updateSettings({ user: users.filter((u) => u.id !== user?.id).map((u) => u.id) });
-        break;
-    }
+    const whoseUsersDictionary = {
```

Comment:
im concerned about this cuz this code override the url params, so users input would be ignored